config/main.yaml

```diff
@@ -19,11 +19,12 @@ seed: 0
 # Hydra config
 hydra:
   # See: https://hydra.cc/docs/configure_hydra/workdir/
+  # See: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes
   run:
-    dir: ${user.logdir.root}/${now:%Y-%m-%d_%H-%M-%S}
+    dir: ${user.logdir.root}/${oc.env:SLURM_JOB_ID,local}/${now:%Y-%m-%d_%H-%M-%S_%f}
```

Comment:
So how exactly does this work if not using Slurm? SLURM_JOB_ID will just be an empty string, or will there be a crash with undefined variable?